wordpress-mobile/WordPress-Android
# Issue/7363 plugin detail row

### Fix
Update the layout of the plugin detail version row including the changing height and button style as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7363.  The tap target of the version text was also updated to fill the height of the row (i.e. card).  See the screenshots and animation below for illustration.

![plugin_directory_row_01](https://user-images.githubusercontent.com/3827611/37182099-07d38a14-22ed-11e8-98e7-57c95ef7d892.png)

![plugin_directory_row_02](https://user-images.githubusercontent.com/3827611/37182102-0aaeb312-22ed-11e8-98ea-b89d5714874d.png)

![plugin_detail_row_flat_height](https://user-images.githubusercontent.com/3827611/37181504-87cf45da-22ea-11e8-96cd-c888a4c70ad2.gif)

The tap target of the plugin view on the plugin directory screen was also updated to be symmetrical.  See the screenshots below for illustration.

![plugin_directory_item](https://user-images.githubusercontent.com/3827611/37182034-adb3f23a-22ec-11e8-9bf3-b5a199c84525.png)

### Test
1. Go to ***Sites*** tab.
2. Tap ***Plugins*** item under ***Configuration*** section.
3. Long-press plugin.
4. Notice symmetrical background.
5. Tap uninstalled plugin.
6. Notice version row layout.
7. Tap enabled ***Install*** button.
8. Notice progress spinner.
9. Notice version row height remains the same.
10. Notice ***Installed*** button.
11. Notice version row height remains the same.
